modules/auxiliary/scanner/portscan/ack.rb

```diff
@@ -43,16 +45,26 @@ def run_batch_size
   end
 
   def run_batch(hosts)
-    open_pcap
-
-    pcap = self.capture
 
     ports = Rex::Socket.portspec_crack(datastore['PORTS'])
-
     if ports.empty?
       raise Msf::OptionValidateError.new(['PORTS'])
     end
 
+    jitter_value = datastore['JITTER'].to_i
```

Comment:
Similar to my previous comment, but all of this stuff that you've added related to sanity checking `JITTER` and `DELAY` should go in the `setup` method of the Scanner mixin, that way it happens automatically in any Scanner module.